go.mod

```diff
@@ -1,18 +1,19 @@
 module github.com/openshift/sandboxed-containers-operator
 
-go 1.21
+go 1.22.7
```

Comment:
This is the reason of the build is failing... The CI environment should update go toolchain or set GOTOOLCHAIN=auto, that will download automatically to satisfy this minimal requirement.

Reply:
I prefer to bump the go toolchain manually for now and keep CI improvements for the next cycle.

https://github.com/openshift/release/pull/59339